types/wordpress__block-editor/components/url-popover.d.ts

```diff
@@ -1,16 +1,16 @@
 import { Popover } from "@wordpress/components";
-import { ComponentType, JSX, ReactNode } from "react";
+import { ComponentProps, ComponentType, JSX, ReactNode } from "react";
 
 declare namespace URLPopover {
-    interface Props extends Popover.Props {
+    type Props = ComponentProps<typeof Popover> & {
```

Comment:
Is there a reason this was changed from interface to type? Some users might be extending the interface and this would break that.

Reply:
Yes. There is `An interface can only extend an object type or intersection of object types with statically known members.ts(2312)` error with `interface`

Reply:
Okay ignore this then :)